Formula/python@3.9.rb

```diff
@@ -4,7 +4,7 @@ class PythonAT39 < Formula
   url "https://www.python.org/ftp/python/3.9.0/Python-3.9.0.tar.xz"
```

Comment:
I think `3.9.1` is out already, would you care to try it?

Reply:
It's still a release candidate https://www.python.org/ftp/python/3.9.1/ with it scheduled to be released on December 11 https://www.python.org/dev/peps/pep-0596/

Reply:
Ah okay, we should fix the livecheck then

Reply:
No, only the rc1:

https://www.python.org/dev/peps/pep-0596/
> 3.9.1 final: Friday, 2020-12-11

Reply:
Not sure how we usually handle these `livecheck` issues. At first glance, maybe we can use https://www.python.org/downloads/source/ with a regex that matches `Latest Python 3 Release - Python 3.9.\d+`